pyproject.toml

```diff
@@ -28,17 +28,17 @@ keywords = [
 exclude = ["tests/"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-Flask = "^1.1.2"
-Flask-SQLAlchemy = "^2.4.1"
-python-decouple = "^3.3"
-SQLAlchemy = "^1.3.16"
+python = ">=3.6.2,<4.0"
+Flask = ">=1.1.2"
+Flask-SQLAlchemy = "^2.5.1"
+python-decouple = "^3.4"
+SQLAlchemy = "^1.4.23"
```

Comment:
It looks like this change breaks the Poetry dev environment. Maybe [related to this issue](https://github.com/python-poetry/poetry/issues/4402). But `poetry install` is not working here, at least (crashing with the error from the issue).

I would ask: why do we need to update SQLAlchemy and how it relates to offering support to Flask 2? Maybe this is a completely different issue to address.